### PR TITLE
EDU-1466: Updates the control-v1.yaml with FCM properties

### PR DESCRIPTION
**Pull request by franrob-projects**
This PR adds the following properties to the control-v1.yaml:

- `fcmServiceAccount`
- `fcmProjectId`

- [x] Version has been incremented
- [x] Tested for errors with Spectral
- [x] Optional: Load into SwaggerHub and check for errors
- [x] Ensure the document renders under ReDoc (see README for instructions on how to test locally)

[EDU-1466: Update control-v1 with FCM properties ](https://ably.atlassian.net/browse/EDU-1466)

### CODE REVIEW

**Review by GregHolmes — APPROVED** [review]

**Review by m-hulbert — CHANGES_REQUESTED** [review]
I think these properties need to be in the PATCH call too, no?

**Comment by franrob-projects** [comment]
> I think these properties need to be in the PATCH call too, no?

@m-hulbert, please see fixup! 09e8d755972e7ca87f5a48de271ddfcd103e5346

File: definitions/control-v1.yaml [hunk 1]
```diff
@@ -4390,6 +4390,26 @@ TOfReTlUQzgpXRW5h3n2LVXbXQhPGcVitb88Cm2R8cxQwgB1VncM8yvmKhREo2tz
...
+        fcmServiceAccount:
+          type: string
+          nullable: true
+          description: Used to specify the Firebase Cloud Messaging(FCM) service account credentials used for authentication and enabling communication with FCM to send push notifications to devices.
+          example: "service_account"
+        fcmProjectId:
+          type: string
+          nullable: true
+          description: The unique identifier for the Firebase Cloud Messaging(FCM) project. This ID is used to specify the Firebase project when configuring FCM or other Firebase services.
+          example: "notional-armor-405018"
```
[the remark this answers is not in the document]
Comment (franrob-projects) [reply definitions/control-v1.yaml:4412]: Thanks :)

[the remark this answers is not in the document]
Comment (franrob-projects) [reply definitions/control-v1.yaml]: @m-hulbert, please see: 3276e0a407d80ea81aadb045e0445a763df9ea58

**Review by m-hulbert — APPROVED** [review]
Rebase and good to go

**Pull request merged**